### ISSUE TITLE
Question preview is ugly

### ISSUE THREAD

**Issue by akdor1154** [nathansherburn/kusema#35]
the question/answer composition form is very cluttered, we should fix it - perhaps previews on the side, and hidden by default on mobile?

Also, the chips in the preview are to the side of the topic autocomplete (they should be below/above probably, or at least the same as the topic one).

Also, the font used for the topic autocomplete is smaller than the group autocomplete.

**Comment by nathansherburn** [comment]
I agree - previews on the side and hidden by default on mobile sounds like a nice solution.

We'll look at fixing the rest once all the functionality is there
